iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -2179,10 +2179,29 @@ def transform_index(
         src_to_target = {
             src: self.target_shape[src_shape.index(src)] for src in src_shape
         }
+        # If `vector_shapes`` is unknown assume we need to switch the strides.
+        if self.vector_shapes is None:
```

Comment:
Isn't there a pass that enforces that every operator has vector_shapes? When would you encounter this situtation where vector_shapes is None?

Reply:
yeah that would be great thanks